extensions/vscode/src/utils/names.ts

```diff
@@ -82,3 +52,45 @@ export function contentRecordNameValidator(
     return undefined;
   };
 }
+
+/**
+ * Creates a semi-unique configuration name from a content title.
+ *
+ * @param title The title of the content to create a filename from
+ * @param existingNames [[]] An array of existing configuration names to ensure
+ *   uniqueness
+ * @returns A filename that is safe to use in the filesystem with a unique 4
+ * character ending to avoid Git conflicts.
+ */
+export function newConfigFileNameFromTitle(
+  title: string,
+  existingNames: string[] = [],
+): string {
+  const filename = filenamify(title, {
+    replacement: "-",
```

Comment:
```suggestion
    replacement: "_",
```
I tend to think of an underscore replacing unwanted characters rather than dash...

Here's my example:
- title: My title for \\this && that / or such
- w/ dashes: `My title for -this && that - or such-FQDE`
- w/ underscores: `My title for _this && that _ or such-FQDE`

Reply:
We used `-` previously before the numbers which I why I used the hyphen here. I could go either way it all depends on if there is a standard for me - like with Python module naming using underscores.

I'll bring it up in standup for a quick check how others are feeling about it.